#!/usr/bin/env python
PACKAGE="riptide_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()


#        Name               Type      Level  Description                    Default     Min         Max
gen.add( "Volume",          double_t, 1,     "Robot Volume",                0,          0,          0.1)
gen.add( "Buoyancy_X_POS",  double_t, 1,     "Buoyancy X Pos.",             0,          -1,         1)
gen.add( "Buoyancy_Y_POS",  double_t, 1,     "Buoyancy Y Pos.",             0,          -1,         1)
gen.add( "Buoyancy_Z_POS",  double_t, 1,     "Buoyancy Z Pos.",             0,          -1,         1)
                 # PkgName  #NodeName         #Prefix for generated .h include file, e.g. ParametersConfig.py
exit(gen.generate(PACKAGE, "thruster_controller", "VehicleProperties"))
